.github/workflows/update_collector_and_scanner.yaml

```diff
@@ -0,0 +1,95 @@
+name: Update collector and scanner version
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 5 * * 1
+jobs:
+  update-collector:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: master # this is our target branch
+        fetch-depth: 0 # we need to fetch all branches
+    - name: Checkout collector repo
+      uses: actions/checkout@v3
+      with:
+        repository: stackrox/collector
+        path: deps/collector
+        fetch-depth: 0 # we need to fetch tags
+    - name: Update version
+      run: make -sC deps/collector tag | tee COLLECTOR_VERSION
+    - name: Create Pull Request
+      id: cpr
+      uses: peter-evans/create-pull-request@v4
+      with:
+        token: '${{ secrets.ROBOT_ROX_GITHUB_TOKEN }}'
+        commit-message: Update COLLECTOR_VERSION
+        committer: RoxBot <roxbot@stackrox.com>
+        author: RoxBot <roxbot@stackrox.com>
+        branch: update_collector_version
+        signoff: false
+        delete-branch: true
+        title: Update COLLECTOR_VERSION
+        body: |
+          Update COLLECTOR_VERSION to latest master version
+        labels: |
+          ci-all-qa-tests
+          dependencies
+        assignees: janisz
+        reviewers: RTann
+        team-reviewers: |
+          collector-team
+          backend-dep-updaters
+        draft: false
+    - name: Enable Pull Request Automerge
+      if: steps.cpr.outputs.pull-request-operation == 'created'
+      uses: peter-evans/enable-pull-request-automerge@v2
+      with:
+        token: '${{ secrets.ROBOT_ROX_GITHUB_TOKEN }}'
+        pull-request-number: '${{ steps.cpr.outputs.pull-request-number }}'
+        merge-method: squash
+  update-scanner:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: master # this is our target branch
+        fetch-depth: 0 # we need to fetch all branches
+    - name: Checkout scanner repo
+      uses: actions/checkout@v3
+      with:
+        repository: stackrox/scanner
+        path: deps/scanner
+        fetch-depth: 0 # we need to fetch tags
+    - name: Update version
+      run: make -sC deps/scanner tag | tee SCANNER_VERSION
+    - name: Create Pull Request
+      id: cpr
+      uses: peter-evans/create-pull-request@v4
+      with:
+        token: '${{ secrets.ROBOT_ROX_GITHUB_TOKEN }}'
+        commit-message: Update SCANNER_VERSION
+        committer: RoxBot <roxbot@stackrox.com>
+        author: RoxBot <roxbot@stackrox.com>
+        branch: update_scanner_version
+        signoff: false
+        delete-branch: true
+        title: Update SCANNER_VERSION
+        body: |
+          Update SCANNER_VERSION to latest master version
+        labels: |
+          ci-all-qa-tests
+          dependencies
+        assignees: janisz
+        reviewers: RTann
+        team-reviewers: |
+          backend-dep-updaters
```

Comment:
I just came here to write this comment: please add scanner or data shepherds team as reviewers.

Reply:
There is no scanner team so I added devs from `@scanner-team` slack group and data shepherds.